src/elm/Routes/TalkToUsPage.elm

```diff
@@ -9,36 +9,35 @@ import Types exposing (..)
 
 talkToUsPage : Model -> Html Msg
 talkToUsPage model =
-    div [ class "ma4" ]
+    div [ class "ma4 ml5-ns" ]
         [ div [ class "gray f3 b" ] [ text "Talk to us" ]
-        , div [ class "gray fw1 mt3 mb3" ] [ text "Whether you know what you service you need and want to book an appointment or need more advice there are a three ways to get in touch. " ]
-        , div [ class "purple-line" ] []
-        , div [ class "tc mt3 bb b--silver pb3" ]
-            [ div [ class " purple f3" ] [ text "Come to a centre" ]
-            , div [ class "mt3 mb3" ]
-                [ text "See a list of our centres "
-                , a
-                    [ class "no-underline b purple", href "http://womenscentre.org/who-we-are/our-centres" ]
-                    [ text "here" ]
+        , div [ class "gray fw1 mt3 mb3 mb4-ns w-50-l" ] [ text "Whether you know what you service you need and want to book an appointment or need more advice there are a three ways to get in touch. " ]
+        , div [ class "purple-line mb3" ] []
+        , a [ href "http://womenscentre.org/who-we-are/our-centres", class "tc db dn black mb3-l" ]
```

Comment:
don't think the `dn` does anything here (it's overwritten by the `db`)

Same elsewhere. Putting this comment as a reminder for both of us